### PR TITLE
Loan interest

### PR DESCRIPTION
**Pull request by JaneMansell**

### CODE REVIEW

**Comment by JaneMansell** [comment]
Conflicts resolved.

File: newbank/server/Display.java [hunk 1]
```diff
@@ -21,13 +21,27 @@ public Display(){
...
+        return "(Customer Names and Account Names are case sensitive. Example: Main)";
+    }
+
+    public String availableCommands(){
+        String s = "" + "\n" + introMessage() + "\n";
+        for(String command : commands) {
+            s += command + "\n";
+        }
+        s += endMessage() + "\n";
+        return s;
```
Comment (ln427) [inline newbank/server/Display.java:44]: This got duplicated with other PR ?

File: newbank/server/Customer.java [hunk 2]
```diff
@@ -130,8 +130,8 @@ private Loan findLoan(Customer customer, List<Loan> loanList){
 		return null;
 	}
 
-	public boolean loan(Customer loanee, double amount) {
-		Loan newLoan = new Loan(this, loanee, amount);
+	public boolean loan(Customer loanee, double amount, double interest) {
+		Loan newLoan = new Loan(this, loanee, amount*(1+interest/100));
```
Comment (ln427) [inline newbank/server/Customer.java:134]: I think this should not be here `amount*(1+interest/100)` 

The calculation above should be inside `Loan` class. Maybe can be part of the constructor -> more encapsulation

File: newbank/server/Display.java (hunk 1 above)
Comment (JaneMansell) [reply newbank/server/Display.java:44]: Yes I don't know why it added this commit to this branch. It is already in the other branch.

File: newbank/server/Customer.java (hunk 2 above)
Comment (JaneMansell) [reply newbank/server/Customer.java:134]: I will have a look at seeing if I can do this.

**Pull request closed without merging**